cpp/ql/src/change-notes/2024-07-23-using-enum-declaration.md

```diff
@@ -0,0 +1,4 @@
+---
+category: minorAnalysis
+---
+* A `UsingEnumDeclarationEntry` class has been added for C++ `using enum` declarations. As part of this, synthesised `UsingDeclarationEntry`s are no longer emitted for individual enumerators of the referenced enumeration.
```

Comment:
Change notes should be in American English.

```suggestion
* A `UsingEnumDeclarationEntry` class has been added for C++ `using enum` declarations. As part of this, synthesized `UsingDeclarationEntry`s are no longer emitted for individual enumerators of the referenced enumeration.
```

Reply:
@jketema I've noted this in https://github.com/github/codeql/pull/17048.